src/test/java/com/hazelcast/commandline/member/MemberCommandLineTest.java

```diff
@@ -0,0 +1,193 @@
+    public void test_start_withPort() throws IOException, ClassNotFoundException, InterruptedException {
+        String port = "9898";
+        memberCommandLine.start(null, null, port, null,
+                false, null, null);
+        assertTrue(memberCommandLine.getProcessOutput().anyMatch(out ->
+                out.contains(port + " is " + LifecycleEvent.LifecycleState.STARTED.toString())));
+    }
+
+    @Test
+    public void test_start_withJVMOptions() throws IOException, ClassNotFoundException, InterruptedException {
+        List<String> javaOptions = new ArrayList<>();
+        String option = "-Xmx128m";
+        javaOptions.add(option);
+        memberCommandLine.start(null, null, null, null,
+                false, null, javaOptions);
+        String output = runCommand("jps -v");
+        String memberOutput = "";
+        for (String line : output.split("\n")) {
+            if (line.contains(HazelcastMember.class.getSimpleName())) {
+                memberOutput = line;
+            }
+        }
+        assertTrue(memberOutput.contains(option));
+    }
+
+    @Test
+    public void test_stop() throws IOException, ClassNotFoundException, InterruptedException {
+        memberCommandLine.start(null, null, null, null,
+                false, null, null);
+        String processUniqueID = captureOut().replace("\n", "");
+        int pid = memberCommandLine.getProcessMap().get(processUniqueID).getPid();
+        memberCommandLine.stop(processUniqueID);
+        assertTrue(!getRunningJavaProcesses().contains(String.valueOf(pid)));
+    }
+
+    @Test
+    public void test_list() throws ClassNotFoundException, IOException, InterruptedException {
+        memberCommandLine.start(null, null, null, null,
+                false, null, null);
+        String processUniqueId1 = captureOut().replace("\n", "");
+        resetOut();
+        memberCommandLine.start(null, null, null, null,
+                false, null, null);
+        String processUniqueId2 = captureOut().replace("\n", "");
+        resetOut();
+        memberCommandLine.list();
+        String out = captureOut();
+        assertTrue(out.contains(processUniqueId1));
+        assertTrue(out.contains(processUniqueId2));
+    }
+
+    @Test
+    public void test_logs() throws IOException, ClassNotFoundException, InterruptedException {
+        String groupName = "member-command-line-test";
+        startMemberWithConfigFile();
+        String processUniqueId = captureOut().replace("\n", "");
+        resetOut();
+        //await for the logs to be created
+        TimeUnit.SECONDS.sleep(5);
+        assertTrue(Files.exists(Paths.get(HazelcastCommandLine.HAZELCAST_HOME
+                + "/" + processUniqueId + "/logs/hazelcast.log")));
+        memberCommandLine.logs(processUniqueId, 1000);
+        assertTrue(captureOut().contains(groupName));
+    }
+
+    @Test
+    public void test_logs_withLineCount() throws IOException, ClassNotFoundException, InterruptedException {
```

Comment:
Minor: You can consider not adding the prefix "test_". You have it in every test method, so it does not seem very useful.